### DISCUSSION TITLE
Compact way to specify DataGrid column header tooltip?

### DISCUSSION THREAD

**Discussion by wgraaf** [AvaloniaUI/Avalonia#14785] [Q&A] [answered]
DataGrid column headers can be made to show a tooltip with:

```
        <DataGridTextColumn Binding="{Binding cellValue}">
          <DataGridTextColumn.HeaderTemplate>
            <DataTemplate>
                <Label Content="header text" ToolTip.Tip="tooltip text"/>
            </DataTemplate>
          </DataGridTextColumn.HeaderTemplate>
        </DataGridTextColumn>
```
Is there a less verbose way to specify the tooltip?

**Comment by maxkatz6** [comment, marked as answer]
No, I don't think you can get less verbose than that.

**Comment by timunie** [comment]
A Style for DataGridHeader could also work, given you have property to bind to.

**Comment by wgraaf** [comment]
How can each column get its own tooltip using a DataGridHeader style?
By binding a to collection going horizontally across the columns?
